Source: Johannes Nixdorf <mixi@shadowice.org>
Upstream: yes, 1.6-current (SVN: r9937)
Reason: A wrong check in configure.in caused a directory called "Resources" to be installed to /
        when no other language files were installed.

--- cups-1.5.0/configure.in.orig	2011-08-30 19:18:18.338581296 +0000
+++ cups-1.5.0/configure.in	2011-08-30 19:18:31.982580723 +0000
@@ -56,7 +56,7 @@
 			LANGFILES="$LANGFILES templates/$lang/header.tmpl"
 		fi
 	done
-elif test "x$CUPS_BUNDLEDIR" != ""; then
+elif test "x$CUPS_BUNDLEDIR" != "x"; then
 	INSTALL_LANGUAGES="install-langbundle"
 	UNINSTALL_LANGUAGES="uninstall-langbundle"
 fi
